diff --git CMakeLists.txt CMakeLists.txt
index 4cc6637..4718576 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -11,13 +11,6 @@ option(BUILD_SHARED_LIBS "Build shared libraries (DLLs)." OFF)
 
 option(gmock_build_tests "Build all of Google Mock's own tests." OFF)
 
-# A directory to find Google Test sources.
-if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/gtest/CMakeLists.txt")
-  set(gtest_dir gtest)
-else()
-  set(gtest_dir ../gtest)
-endif()
-
 # Defines pre_project_set_up_hermetic_build() and set_up_hermetic_build().
 include("${gtest_dir}/cmake/hermetic_build.cmake" OPTIONAL)
 
